### PR TITLE
Import notify-management space

### PR DESCRIPTION
**Pull request by jskinne3**
## Description

Importing the `notify-management` space under Terraform management, with a `prevent_destroy` on it

## Deployment

⚠️ Only deploy this after all other currently outstanding Terraform PRs have been deployed and verified, namely #922, #923, #925, #926, and #928

### CODE REVIEW

File: terraform/bootstrap/imports.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# The notify-management space contains resources that don't belong in our
+# deployed environment spaces (notify-sandbox, etc.) including:
+#   * the bucket that holds Terraform state
+#   * supplemental service brokers, letting cloud.gov provision AWS services
+#   * some service accounts used for deployment
+
+import {
+  to = cloudfoundry_space.notify-management
+  id = "a022fabe-056c-4329-9d4f-3f7a67442d36"
+}
```
Comment (ccostino) [inline terraform/bootstrap/imports.tf:10]: Calling out the fact that with the way this is done, which is very straightforward and makes sense, I'm assuming this'll be the approach for the other spaces that aren't currently under Terraform management (e.g., `notify-staging`)?

**Review by ccostino — COMMENTED** [review]
Thanks, @jskinne3!

Noting here that this'll require a rebase with `main` once the first batch of PRs are merged when we're ready.

File: terraform/bootstrap/imports.tf (hunk 1 above)
Comment (jskinne3) [reply terraform/bootstrap/imports.tf:10]: Yup! This is the technique I think we should use to bring other spaces into Terraform

**Comment by ccostino** [comment]
@jskinne3 checking in on this PR as well!

**Comment by jskinne3** [comment]
Because of the difficulty we encountered importing this space and then running `terraform destroy` I think this is a bad idea, at least for now. I am closing this PR.

**Pull request closed without merging**